lib/defaults.js

```diff
@@ -2,260 +2,111 @@
 
 module.exports = {
   extends: 'airbnb-base/legacy',
-  'env': {                           // http://eslint.org/docs/user-guide/configuring.html#specifying-environments
-    'browser': true,                 // browser global variables
-    'node': true,                    // Node.js global variables and Node.js-specific rules
-    'es6': true
+
+  env: {                           
+    browser: true, // Browser global variables
+    node: true, // Node.js global variables and Node.js-specific rules
+    es6: true
   },
-  'plugins': [
+
+  plugins: [
     'virtru-lint'
   ],
 
-/*@TODO: Decide upon these later
-  'ecmaFeatures': {
-    'arrowFunctions': true,
-    'blockBindings': true,
-    'classes': true,
-    'defaultParams': true,
-    'destructuring': true,
-    'forOf': true,
-    'generators': false,
-    'modules': true,
-    'objectLiteralComputedProperties': true,
-    'objectLiteralDuplicateProperties': false,
-    'objectLiteralShorthandMethods': true,
-    'objectLiteralShorthandProperties': true,
-    'spread': true,
-    'superInFunctions': true,
-    'templateStrings': true,
-    'jsx': true
-  },
-*/
-  'rules': {
+  rules: {
     /**
-     * Node
+     * See http://eslint.org/docs/rules/
+     *
+     * Values (see http://eslint.org/docs/user-guide/configuring):
+     *  0 = Rule turned off
+     *  1 = Warn
+     *  2 = Error (causes non-zero exit code)
+     *
+     * Docs for a rule, %s, are found at http://eslint.org/docs/rules/%s
+     *  Example: http://eslint.org/docs/rules/no-shadow
      */
-    'global-require': 0,
 
     /**
-     * Strict mode
+     * Node
      */
-    //@TODO: We aren't using babel at the moment.
-    //babel inserts 'use strict'; for us
-    //'strict': [2, 'never'],          // http://eslint.org/docs/rules/strict
-
-
+    'global-require': 0,
 
     /**
      * Virtru Specific Rules
      */
-    'virtru-lint/virtru-func-names': 2,
+    'virtru-lint/virtru-func-names': 1,
```

Comment:
This rule is becoming less relevant with our use of arrow functions in most repos, and regardless of that I feel it should result in a warning and not an error.